### PR TITLE
Edited the example tf files regarding the issue #39

### PR DESCRIPTION
**Pull request by gekios**
# What does this PR?

I modified the example tf files in order to be inline with `sles12sp3.tf` creating 4 VMs and 2 disks on each on of them for issue #39

### CODE REVIEW

File: examples/libvirt/archlinux.tf [hunk 1]
```diff
@@ -11,28 +11,47 @@ module "archlinux" {
...
   vcpu      = 1
-  count     = 1
+  count     = "${var.count}"
   cloudinit = "${module.cloudinit.cloudinit_id}"
 
   network_interface {
     network_name = "default"
   }
 
+  // OS image
```
Comment (MalloZup) [inline examples/libvirt/archlinux.tf]: this comment with the other ones is not giving an extra info

File: examples/libvirt/archlinux.tf [hunk 2]
```diff
@@ -11,28 +11,47 @@ module "archlinux" {
...
   disk {
     volume_id = "${element(libvirt_volume.archlinux_disk.*.id, count.index)}"
   }
 
+  // DISK
```
Comment (MalloZup) [inline examples/libvirt/archlinux.tf]: this comment is not adding value

File: examples/libvirt/centos.tf [hunk 3]
```diff
@@ -11,29 +11,64 @@ module "cloudinit" {
...
   vcpu      = 1
-  count     = 1
+  count     = "${var.count}"
   cloudinit = "${module.cloudinit.cloudinit_id}"
 
   network_interface {
     network_name = "default"
   }
 
+  // OS image
```
Comment (MalloZup) [inline examples/libvirt/centos.tf]: same as all others

**Comment by MalloZup** [comment]
@gekios  check out why travis is failing, since this is a regression the master is green thx ! :tulip:

**Comment by gekios** [comment]
@MalloZup  thank you for the feedback, I didn't expect it this late tbh. the issue was that I hadn't run 'terraform fmt' so I ll run it and commit.  

Regarding the "Disk" comments, the first disk (eg `libvirt_volume.archlinux_disk`) is the disk for the "OS" while the `osd disks` are the "data" ones, right? I just want to clarify this because I am not sure I get it right.

**Comment by MalloZup** [comment]
@gekios in opensource there is no time :) people around the world start when you sleep :smile: 
Anyway looks good, we could clarify  the `DISK` comment and we can modify with `empty DISK data` and the other we could write `DISK containing Operationg Systems`. This could be good.

**Comment by gekios** [comment]
@MalloZup Cool thanks for the update. I will edit the comments as you suggested when I go back home from work :)

**Comment by gekios** [comment]
@MalloZup Good morning Dario. I edited the comments as you requested yesterday, I just didn't want to spam you for this outside working hours again. Please take a look when you have some time and let me know if there is any issue :)

**Pull request merged**